web/docs/understand-vast/architecture/plugins.md

```diff
@@ -53,39 +53,47 @@ analyzer plugin is a refinement of the [component plugin](#component).
 The reader plugin adds a new format to parse input data, such as JSON (ASCII) or
 PCAP (binary).
 
-Reader plugins automatically add the subcommands `vast import <format>`.
+Reader plugins automatically add the subcommand `vast import <plugin name>`.
 
 ### Writer
 
 The writer plugin adds a new format to print data, such as JSON (ASCII) or PCAP
 (binary).
 
-Writer plugins automatically add the subcommands `vast export <format>`.
+Writer plugins automatically add the subcommand `vast export <plugin name>`.
 
 ### Query Language
 
 A query language plugin adds an alternative parser for a query expression. This
 plugin allows for replacing the query *frontend* while using VAST as *backend*
 execution engine.
 
-For example, you could write a SQL plugin that takes an expression like
-`SELECT * FROM zeek.conn WHERE id.orig_h = "1.2.3.4"` and executes it on
-historical data or runs it as live query.
+For example, you could write a SQL plugin that takes an expression like `SELECT
+* FROM zeek.conn WHERE id.orig_h = "1.2.3.4"` and executes it on historical data
```

Comment:
The splitting of the backtick expression might cause rendering issues, no?

Reply:
It doesn't.

Reply:
Apparently that depends on where you look at the file, on GitHub it renders correctly but on Docusaurus it renders wrongly. I changed it.